src/pages/Account/Account.tsx

```diff
@@ -29,23 +81,23 @@ const Account = () => {
           <div className="account-page__text-content">
             <InputBox
               label={"First Name"}
-              inputPlaceholder="Jugraj"
+              inputPlaceholder={capitalizeFirstCharacter(firstName)}
               inputType={"text"}
               handleInput={handleInput}
             />
             <div className="account-page__text-content--box"></div>
 
             <InputBox
               label={"Last Name"}
-              inputPlaceholder="Singh"
+              inputPlaceholder={capitalizeFirstCharacter(lastName)}
               inputType={"text"}
               handleInput={handleInput}
             />
             <div className="account-page__text-content--box"></div>
 
             <InputBox
               label={"Email"}
-              inputPlaceholder="example@gmail.com"
+              inputPlaceholder={email}
               inputType={"email"}
               handleInput={handleInput}
             />
```

Comment:
do we want these to be inputboxes? We are just displaying the information so we can just use P tags now? unless we are going to allow them to change their info but at this stage we won't get time to add that feature.